packages/admin-ui-docs/src/components/CodeSection/index.tsx

```diff
@@ -0,0 +1,34 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+
+import { PageSection } from '../PageSection'
+
+import CodeBlock from '../../theme/CodeBlock'
+
+export interface CodeSectionProps {
+  children: ReactNode
+  code: string
+  columnsCount: number
+  direction: 'column' | 'row'
+  id: string
+  isLive: boolean
+  isNoInline: boolean
+  layout: 'grid' | null
+  title: string
+}
+
+export function CodeSection(props: CodeSectionProps) {
+  return (
+    <PageSection id={props.id} title={props.title} direction={props.direction}>
+      <CodeBlock
+        className="language-jsx"
```

Comment:
Is it a problem having this always set to `jsx`? I know that we have some use cases where we use only `ts` and others with `bash`, but I don't know if this code influences these cases

Reply:
It does not make much difference, but i will try to pass the language as prop...